doc/source/ecosystem.md

```diff
@@ -0,0 +1,27 @@
+(ecosystem)=
+
+# Ecosystem
+
```

Comment:
Maybe not here, but somewhere; should we refer to demcompare as that's still used quite much? Sorry if it already says somewhere.

Reply:
Good point! Added for now, CNES/CS can orient us on how they'd like demcompare to show up in the doc.